src/lib/utils/renderWithMantine.tsx

```diff
@@ -0,0 +1,8 @@
+import { MantineProvider } from "@mantine/core";
```

Comment:
Let's create a new directory in here `src/lib/utils/test-utils/` where we put any functions we use exclusively for running tests